Source/NSCollectionView.m

```diff
@@ -629,6 +709,7 @@ - (id) initWithCoder: (NSCoder *)aCoder
           decode_NSUInteger(aCoder, &_maxNumberOfColumns);
           [aCoder decodeValueOfObjCType: @encode(BOOL) at: &_isSelectable];
           [self setBackgroundColors: [aCoder decodeObject]]; // decode color...
+          [self setCollectionViewLayout: [aCoder decodeObject]];
```

Comment:
Most likely there are no gorm files for this class, but normally we would change the version of the class and make the decoding version dependent.

Reply:
Keysight has been using this class for a while, not from Gorm, though.  I just updated it since it is harmless to do so.